### ISSUE TITLE
pydeck: Ability for rendered HTML documents to load without internet access

### ISSUE THREAD

**Issue by bandersen23** [uber/deck.gl#4290]
#### Target Use case
Embedding the required javascript needed for deck.gl to work within pydeck would enable ability to dynamically create maps in a situation where you might not have access to the internet.

#### Proposed feature
This could be established fairly easily by creating an argument within `Deck` that would then change the CDN to something local to the package.
https://github.com/uber/deck.gl/blob/4211e5ed777cdaa4412de7f6e6f423cf2f5c08f8/bindings/pydeck/pydeck/io/html.py#L22

The same thing could be done with the CSS referenced as well:
https://github.com/uber/deck.gl/blob/4211e5ed777cdaa4412de7f6e6f423cf2f5c08f8/bindings/pydeck/pydeck/io/templates/index.j2#L5

https://github.com/uber/deck.gl/blob/4211e5ed777cdaa4412de7f6e6f423cf2f5c08f8/bindings/pydeck/pydeck/io/templates/index.j2#L6

#### To Do List 
  - [ ] Add label and assign to milestone 
  - [ ] Coding
  - [ ] Doc update
  - [ ] What’s new update 
  - [ ] Test

**Comment by ajduberstein** [comment]
Thanks for the suggestion! This is a good idea and shouldn't be too hard to implement. Give me a few weeks.

**Comment by bandersen23** [comment]
@ajduberstein Awesome, thank you!

**Comment by ajduberstein** [comment]
Addressed by #4405

**Comment by AhmedThahir** [comment]
Is there a way to add a custom URL for the assets? As you can see, despite the column chart appearing, the mapbox graph is not appearing offline.

![Screen Shot 2023-08-20 at 1 50 20 PM](https://github.com/visgl/deck.gl/assets/94914628/43d3b594-1058-445e-808e-f71e6c409704)

I really like Plotly's implementation of offline export

```
include_plotlyjs: bool or string (default True)
        Specifies how the plotly.js library is included/loaded in the output
        div string.
    
        If True, a script tag containing the plotly.js source code (~3MB)
        is included in the output.  HTML files generated with this option are
        fully self-contained and can be used offline.
    
        If 'cdn', a script tag that references the plotly.js CDN is included
        in the output. HTML files generated with this option are about 3MB
        smaller than those generated with include_plotlyjs=True, but they
        require an active internet connection in order to load the plotly.js
        library.
    
        If 'directory', a script tag is included that references an external
        plotly.min.js bundle that is assumed to reside in the same
        directory as the HTML file. If `file` is a string to a local file path
        and `full_html` is True then
```
